login.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width= , initial-scale=1.0" />
+    <title>login</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="login_style.css" />
+  </head>
+  <body>
+    <header>
+      <a href="/"><img src="/image/logo3x.png" alt="mainlogo" /></a>
+    </header>
+    <main>
+      <form>
+        <div class="eamil">
+          <label for="signup-email">이메일</label>
+          <input
+            id="signup-email"
+            name="email"
+            placeholder="이메일을 입력해주세요"
+          />
+        </div>
+      </form>
+      <form>
+        <div class="password">
+          <label for="signup-password">비밀번호</label>
+          <input
+            id="signup-password"
+            name="password"
+            type="password"
+            placeholder="비밀번호를 입력해주세요"
+          />
+          <button class="password-incon"></button>
+        </div>
+      </form>
+      <button type="submit">로그인</button>
```

Comment:
```suggestion
      <form>
        <div class="eamil">
          <label for="signup-email">이메일</label>
          <input
            id="signup-email"
            name="email"
            placeholder="이메일을 입력해주세요"
          />
        </div>
        <div class="password">
          <label for="signup-password">비밀번호</label>
          <input
            id="signup-password"
            name="password"
            type="password"
            placeholder="비밀번호를 입력해주세요"
          />
         <button class="password-incon"></button>
        </div>
        <button type="submit">로그인</button>
      </form>
```

`form`태그는 한번만 사용하시고 하나의 `form`안에 폼요소들을 넣으셔야합니다 ! 
그리고 `form`태그 내에서 html 요소들의 기본행동이 몇가지 결정되는데 
`form`태그 내부에서 등장하는 가장 첫번째 `button`태그는 `type: submit`으로 결정됩니다. 

`form`요소에서 `type:submit`인 버튼의 역할은 무엇일까요? 
그리고 여러가지 버튼이 있는 경우 내가 원하지 않는 버튼이 `type:submit`이 될 경우 이를 해결하기 위해 어떻게 해야할까요 ?

Reply:
form을 두개썼었네요...ㅎㅎ

form 안에서 type:submit 버튼은 누르게되면 값이 전송되는 역할을 가지고 있는걸로 알고있습니다..!!
그리고 type은 submit 말고 button이나 reset으로 지정해서 타입을 바꾸면 됩니다!!